schnet-graph-network/PyG-SchNetGNN.ipynb

```diff
@@ -2,7 +2,6 @@
  "cells": [
```

Comment:
<p>Can you put the instructions in this one as well? please</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-Pytorch-fork/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>